### PR TITLE
Fixes infos & problems with remember rake mailers

### PR DESCRIPTION
**Pull request by patrickemuller**
- Removes unused gems
- Adds basic preview for remember mailers (/rails/mailers)
- Fixes wrong value for subtotal on producer mailer
- Adds number of quotes per buyer on deliver coordinator mailer
- Removes some confused IFs inside remembers.rake file

### CODE REVIEW

File: app/views/remembers/deliver_coordinator.text.erb [hunk 1]
```diff
@@ -16,9 +16,10 @@ Produtos:
 COLETA
 
 Horário de coleta: entre <%= I18n.l @offer.collect_starts_at %> às <%= I18n.l @offer.collect_ends_at %>.
-Cotistas (<%= @offer.purchases.by_status(PurchaseStatus::PAID).select(:user_id).count %> pessoas):
-<% @offer.purchases.by_status(PurchaseStatus::PAID).each_with_index do |purchase, index| %>
-<%= index + 1 %>) <%= purchase.user.name %> - <%= purchase.user.phone %>
+Cotistas (<%= @purchases.select(:user_id).count %> pessoas):
+<% @purchases.each_with_index do |purchase, index| %>
+  <% quantity = purchase.orders.where(offer_id: @offer.id).sum(:quantity) %>
+  <%= index + 1 %>) <%= purchase.user.name %> (<%= quantity %> cotas)- <%= purchase.user.phone %>
```
Comment (patrickemuller) [inline app/views/remembers/deliver_coordinator.text.erb:22]: Here I added a missing info about how much quotes the user has for this offer.

File: app/views/remembers/producer.text.erb [hunk 2]
```diff
@@ -15,8 +15,8 @@ Horário: <%= I18n.l offer.collect_starts_at - 2.hours, format: :short %> aproxi
 Produtos:
 <%= ActionController::Base.helpers.strip_tags offer.products_description.gsub(/<\/(p|li|ul|br)>/, "\n") %>
 
-SUBTOTAL = <%= ActionController::Base.helpers.number_to_currency offer.value %> x <%= offer.stock %> = <%= ActionController::Base.helpers.number_to_currency offer.stock * offer.value %>
-<% items << offer.stock * offer.value %>
+SUBTOTAL = <%= ActionController::Base.helpers.number_to_currency offer.value %> x <%= offer.orders.sum(:quantity) %> = <%= ActionController::Base.helpers.number_to_currency offer.purchases.sum(:total) %>
+<% items << offer.purchases.sum(:total) %>
```
Comment (patrickemuller) [inline app/views/remembers/producer.text.erb:19]: The math was wrong...

File: Gemfile [hunk 3]
```diff
@@ -67,15 +67,8 @@ group :development, :test do
   gem 'machinist', '~> 2.0'
   gem 'rspec-rails', '~> 3.3.2'
   gem 'awesome_print', '~> 1.6.1', require: false
-  gem 'spring-commands-rspec', '~> 1.0.4'
```
Comment (ssjr) [inline Gemfile:70]: I use

File: Gemfile [hunk 4]
```diff
@@ -67,15 +67,8 @@ group :development, :test do
...
   gem 'thin', '~> 1.6.3'
   gem 'pry-rails'
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
-  # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
```
Comment (ssjr) [inline Gemfile:78]: also...

**Pull request merged**